src/modules/PublicLab.MapModule.js

```diff
@@ -24,8 +24,12 @@ module.exports = PublicLab.MapModule = PublicLab.Module.extend({
      }
 
      _module.blurredLocation = new BlurredLocation(options) ;
-
-     _module.blurredLocation.panMapToGeocodedLocation("placenameInput") ;
+        
+        //check if "google" is defined PLOTS2#4717
+    google
```

Comment:
Does this have to be window.hasOwnProperty('google')?

Reply:
I tested this snippet @jywarren, there's no need to explicitly check the window object, since google gets binded to it when we include the scripts.

Reply:
Hmm, really? Just running this locally i get:

```js
google ? alert('hi') : alert('bye')
> Uncaught ReferenceError: google is not defined
```
Sorry, just trying to be extra sure -- and perhaps I misunderstand how this will be run, so happy to learn!

Reply:
@jywarren I was unable to reproduce this for some reason, `google` always shows up defined on my local. :thinking: 

![peek 2019-02-06 23-57](https://user-images.githubusercontent.com/33557095/52364388-15f8b980-2a6b-11e9-947e-773412959634.gif)

Reply:
is it possible it is actually defined there? Can you try it with, say, a made- up variable name? I think we're watching for when the google library is not included, right?

Reply:
Looks like both approaches solved the issue. :smile: 

![screenshot from 2019-02-07 01-26-12](https://user-images.githubusercontent.com/33557095/52369913-b523ae00-2a77-11e9-8936-fdca26cc6283.png)

Reply:
Oh! Didn't see your comment there since we both posted at the same time. Looking into this.

Reply:
>Can you try it with, say, a made- up variable name?

![screenshot from 2019-02-07 01-34-12](https://user-images.githubusercontent.com/33557095/52370351-ab4e7a80-2a78-11e9-82c7-72cd4782c174.png)